src/rewriter/mkrewrites.py

```diff
@@ -88,7 +89,10 @@ def gen_mk_node(defns, expr):
         return expr.name
     elif isinstance(expr, App):
         args = ",".join(gen_mk_node(defns, child) for child in expr.children)
-        if (expr.op == Op.EXTRACT or expr.op == Op.REPEAT or expr.op == Op.ZERO_EXTEND or expr.op == Op.SIGN_EXTEND or expr.op == Op.ROTATE_LEFT or expr.op == Op.ROTATE_RIGHT or expr.op == Op.INT_TO_BV):
+        if (expr.op == Op.EXTRACT or expr.op == Op.REPEAT or \
```

Comment:
No need to add parenthesis here.

Reply:
do you think we should just use set membership here

Reply:
fixed